spec/features/user_posts_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+RSpec.feature 'User Post Index Page', type: :feature do
```

Comment:
- [ ] Hi @Team! 👋 Fantastic job on incorporating feature tests for the user post index page! 🎉 However, I've noticed that some test cases, as indicated in the screenshot below, are either missing or incorrectly placed in the wrong file. 🤔 Kindly ensure you add the missing ones or relocate the ones placed erroneously from other files to this specific file. 🔄 Your attention to this detail will help maintain a comprehensive and organized testing suite. Much appreciated! 🙌

![Screenshot from 2023-12-05 18-20-46](https://github.com/GhausAliKhan/Blog-app/assets/112261237/60119702-d4bf-444a-af55-3c850b258563)